content/docs/1.5.4/references/networking.md

```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
```

Comment:
Please update the file path using the corresponding version. Also, update the doc for other versions.

https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/chart/values.yaml

Reply:
Done.